### ISSUE TITLE
DatePicker not dismissing unless you force focus on it

### ISSUE THREAD

**Issue by Jahrenski** [telerik/kendo-angular#1504]
## I'm submitting a...
* Bug report 

## Current behavior
Unable to close the date picker's popup by clicking anywhere else unless I force focus the element.

## Expected behavior
I expect the date picker's popup to close without having to force the focus on the element.

## Minimal reproduction of the problem with instructions
[https://plnkr.co/edit/NqoGn3AN4ntvVUlO6nF8](https://plnkr.co/edit/NqoGn3AN4ntvVUlO6nF8)

You need to have the device toolbar activated in the devtools (chrome). Or use a real device
Start Date picker forces the focus onto itself and clicking anywhere else will trigger the blur event that seems to close the popup.

End Date picker does not force the focus and will not close on blur

## Environment

Package versions:
```
+-- @angular/animations@5.2.10
+-- @angular/cdk@5.2.5
+-- @angular/common@5.2.10
+-- @angular/compiler@5.2.10
+-- @angular/compiler-cli@5.2.10
+-- UNMET PEER DEPENDENCY @angular/core@5.2.10
+-- @angular/forms@5.2.10
+-- @angular/http@5.2.10
+-- @angular/material@5.2.5
+-- @angular/platform-browser@5.2.10
+-- @angular/platform-browser-dynamic@5.2.10
+-- @angular/platform-server@5.2.10
+-- @compodoc/compodoc@1.1.2
+-- @ionic-native/background-geolocation@4.7.0
+-- @ionic-native/barcode-scanner@4.7.0
+-- @ionic-native/core@4.7.0
+-- @ionic-native/keyboard@4.7.0
+-- @ionic-native/network@4.7.0
+-- @ionic-native/screen-orientation@4.7.0
+-- @ionic-native/splash-screen@4.7.0
+-- @ionic-native/sqlite@4.7.0
+-- @ionic-native/status-bar@4.7.0
+-- @ionic/app-scripts@3.1.9
+-- @ionic/storage@2.1.3
+-- @ngx-translate/core@9.1.1
+-- @ngx-translate/http-loader@2.0.1
+-- @progress/kendo-angular-buttons@4.0.0
+-- @progress/kendo-angular-charts@2.0.3
+-- @progress/kendo-angular-dateinputs@2.2.0
+-- @progress/kendo-angular-dialog@3.4.0
+-- @progress/kendo-angular-dropdowns@2.1.0
+-- @progress/kendo-angular-excel-export@1.0.7
+-- @progress/kendo-angular-grid@3.0.0
+-- @progress/kendo-angular-inputs@2.2.0
+-- @progress/kendo-angular-intl@1.4.0
+-- @progress/kendo-angular-l10n@1.1.0
+-- @progress/kendo-angular-layout@2.2.1
+-- @progress/kendo-angular-popup@2.2.0
+-- @progress/kendo-angular-scrollview@1.0.11
+-- @progress/kendo-angular-upload@3.0.3
+-- @progress/kendo-data-query@1.2.2
+-- @progress/kendo-drawing@1.5.2
+-- @progress/kendo-theme-default@2.50.0
+-- angular2-moment@1.8.0
+-- azure-mobile-apps-client@2.0.1
+-- cordova-android@7.1.0
+-- cordova-ios@4.5.4
+-- cordova-plugin-camera@4.0.3
+-- cordova-plugin-device@2.0.2
+-- cordova-plugin-ionic-webview@1.2.0
+-- cordova-plugin-ionic-wkkeyboard@1.1.15
+-- cordova-plugin-network-information@2.0.1
+-- cordova-plugin-splashscreen@5.0.2
+-- cordova-plugin-statusbar@2.4.2
+-- cordova-plugin-whitelist@1.3.3
+-- cordova-sqlite-storage@2.3.0
+-- cordova-windows@6.0.0
+-- crypto-js@3.1.9-1
+-- esprima@4.0.0
+-- hammerjs@2.0.8
+-- ionic-angular@3.9.2
+-- ionicons@3.0.0
+-- jquery@3.3.1
+-- localforage@1.7.1
+-- ng-inline-svg@6.1.0
+-- ng2-ckeditor@1.2.0
+-- ng2-logger@2.0.35
+-- ng2-simple-timer@1.3.3
+-- ngx-barcode@0.2.4
+-- node-uuid@1.4.8
+-- phonegap-plugin-barcodescanner@7.1.1
+-- rxjs@5.5.10
+-- sass-lint@1.12.1
+-- sw-toolbox@3.6.0
+-- tslint@5.9.1
+-- UNMET PEER DEPENDENCY typescript@2.8.1
`-- zone.js@0.8.26
```

Browser:
- Chrome (desktop) Version 68.0.3415.0 (Official Build) canary (64-bit)

System:
- TypeScript version: 2.8.1
- Node version: v8.1.4 - npm: 5.6.0
- Platform:  Windows

**Comment by ggkrustev** [comment]
Hi @Jahrenski,
it seems that the plunkr works now. Could you provide a runnable demo or a repro steps that could help us to observe the problem locally? Thanks in advance.

**Comment by Jahrenski** [comment]
Updated the main post with the link to the plnkr repro.

**Comment by ggkrustev** [comment]
Hi @Jahrenski,

Thanks for the repro demo. Indeed, the popup won't close on document click when loaded in mobile device. 

In general, the component closes the popup on blur, but hence neither the input or the calendar is focused on open, the popup is left opened. The described behavior is actually intended, because we don't want to show virtual keyboard (on input focus) when popup is toggled.

I do agree that leaving the popup opened is not lead to great U. We will research the case more and will update the thread once we have more information.

**Comment by ggkrustev** [comment]
A fix is released in `v3.4.0-dev.201805210833`. The easiest to try the change is to install the `DEV` release:
`npm i @progress/kendo-angular-dateinputs@dev`

**Comment by ggkrustev** [comment]
Fixed in `v3.4.0`. Please install the latest package to see the changes.

`npm i @progress/kendo-angular-dateinputs@3.4.0`
